src/ProgramBlockEditor.scss

```diff
@@ -73,56 +73,54 @@
     }
 }
 
-button.btn.ProgramBlockEditor__program-deleteAll-button {
+.ProgramBlockEditor__program-deleteAll-button {
     background-color: #4C9990;
     border: none;
     border-radius: 50%;
     box-shadow: none;
     margin-left: 1.5rem;
-    padding: 0.5rem;
 
     &:focus {
         box-shadow: 0 0 0 $c2lc-focus-indicator-width black;
     }
 }
 
-button.btn.ProgramBlockEditor__program-deleteAll-button--pressed {
+.ProgramBlockEditor__program-deleteAll-button--pressed {
```

Comment:
We can remove this style, as this className doesn't exist in ProgramBlockEditor.

Reply:
I went through and got rid of all the ones that either singly or in combination were for the unused `pressed` variant.